consensus-types/blocks/execution_test.go

```diff
@@ -81,6 +81,76 @@ func TestWrapExecutionPayloadHeader_SSZ(t *testing.T) {
 	assert.NoError(t, wsb.UnmarshalSSZ(encoded))
 }
 
+func TestWrapExecutionPayloadCapella(t *testing.T) {
+	data := &enginev1.ExecutionPayloadCapella{GasUsed: 54}
```

Comment:
Personally, I think there's great value to fill in all the fields and test they are not nil